src/java.base/windows/classes/sun/nio/fs/WindowsFileStore.java

```diff
@@ -44,6 +52,8 @@
     private final int volType;
     private final String displayName;   // returned by toString
 
+    private int hashCode = 0;
+
```

Comment:
Good, I think this is safe and equals/hashCode are consistent. I suspect we can extend this to other (maybe all) volume type but it needs a bit more exploration. Minor nit is that you can't need to initialise hashCode to 0.

Reply:
I think this issue was specific to the `DRIVE_FIXED` case so further enhancement can be done under another issue once the information becomes knowd.

Reply:
> I think this issue was specific to the `DRIVE_FIXED` case so further enhancement can be done under another issue once the information becomes knowd.

That's okay with me.